pyQuARC/code/custom_validator.py

```diff
@@ -102,3 +95,106 @@ def bounding_coordinate_logic_check(coordinates_dictionary):
         )
 
         return result
+
+    @staticmethod
+    def presence_check(*field_values):
+        """
+            Checks if one of the field has a value
+        """
+        # At least one of all the fields should have a value
+        # It is basically a OneOf check
+        validity = False
+        value = None
+
+        for field_value in field_values:
+            if field_value:
+                value = field_value
+                validity = True
+
+        return {
+            "valid": validity,
+            "value": value
+        }
+
+    @staticmethod
+    @if_arg
+    def opendap_url_location_check(field_value):
+        # The field shouldn't have a opendap url
+        return {
+            "valid": 'opendap' not in field_value.lower(),
+            "value": field_value
+        }
+
+    @staticmethod
+    @if_arg
+    def user_services_check(first_name, middle_name, last_name):
+        return {
+            "valid": not (first_name.lower() == 'user' and last_name.lower() == 'services' and (not middle_name)),
```

Comment:
What’s this check?

Reply:
this one: https://github.com/NASA-IMPACT/pyQuARC/issues/70